lmdeploy/pytorch/engine/model_agent.py

```diff
@@ -289,7 +289,7 @@ def split(self, split_size: int, block_size: int):
             if overlap:
                 block_end += 1
 
-            block_offsets = self.block_offsets[:, :block_end]
+            block_offsets = self.block_offsets
```

Comment:
@RunningLeon Is this OK?

Reply:
Right. Because here is a bug as mentioned in https://github.com/InternLM/lmdeploy/pull/1652#issuecomment-2151898694